### DISCUSSION TITLE
Bookmarklets Blocked

### DISCUSSION THREAD

**Discussion by HeroLink6** [3kh0/ext-remover#883] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
So, my Chromebook has Bookmarklets blocked by the school, and I don't know if any of the exploits will work for me without bookmarklets. (I also have chrome://version, and chrome://flags blocked.)

**Comment by S-PScripts** [comment]
you can view your chromebook version through the settings app.
also with flags the only bypass there is is the anti auto chromebook update

**Reply by HeroLink6** [reply]
oh, but I don't think that I can get the board type. And the website that is used for the anti-Chromebook update is also blocked.

**Reply by AshtonDavies** [reply]
Try:
caub.netlify.app
caub.vercel.app
etc...

**Reply by S-PScripts** [reply]
As AshtonDavies said, use the non-flag CAUB. 

extra link: https://caub.onrender.com/

**Reply by S-PScripts** [reply]
chrome://system/
check 🔗 CHROMEOS_RELEASE_BOARD or CHROMEOS_RELEASE_BUILDER_PATH or CHROMEOS_RELEASE_DESCRIPTION or platform_identity_name. it will say the board name (mine was octopus)

**Reply by HeroLink6** [reply]
hmm, ok, I will try

**Reply by HeroLink6** [reply]
Would "hatch" be my board name??
hatch-signed-mp-v8keys

**Reply by S-PScripts** [reply]
Yep :) !

**Reply by HeroLink6** [reply]
ok
